### PR TITLE
Finished reviewing module 7

### PR DESCRIPTION
**Pull request by waelkdouh**

### CODE REVIEW

**Comment by pankajagrawal16** [comment]
If possible, please avoid changes to any .md docs file up until module 4. Else it will create alot of conflicts with mkdocs branch

File: docs/aca/07-aca-cron-bindings/index.md [hunk 1]
```diff
@@ -92,50 +105,120 @@ public interface ITasksManager
...
-        return tasksList.ToList();
+        _logger.LogInformation("Mark task with Id: '{0}' as OverDue task", taskModel.TaskId);
+        taskModel.IsOverDue = true;
+        await _daprClient.SaveStateAsync<TaskModel>(STORE_NAME, taskModel.TaskId.ToString(), taskModel);
     }
+}      
+
+```
+
+Make sure you add the following using statements at the the top of the `TasksStoreManager.cs` which is located under the `services` folder of the `TasksTracker.TasksManager.Backend.Api` project
```
Comment (tjoudeh) [inline docs/aca/07-aca-cron-bindings/index.md:156]: `Services` with capital S

File: docs/aca/07-aca-cron-bindings/index.md [hunk 2]
```diff
@@ -92,50 +105,120 @@ public interface ITasksManager
...
+            new JsonStringEnumConverter(),
+            new DateTimeConverter("yyyy-MM-ddTHH:mm:ss")
+        },
+        PropertyNameCaseInsensitive = true,
+        DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull,
+        Encoder = JavaScriptEncoder.UnsafeRelaxedJsonEscaping
+    };
+    var yesterday = DateTime.Today.AddDays(-1);
+
+    var jsonDate = JsonSerializer.Serialize(yesterday, options);
```
Comment (tjoudeh) [inline docs/aca/07-aca-cron-bindings/index.md:128]: @waelkdouh Were you able to test that returned datetime format matches the one used with Newtonsoft and execute the method successfully? Let's make sure that at least some tasks are marked overdue after this change. From the code here it should work as you are specifying the format explicitly.

**Review by tjoudeh — APPROVED** [review]
Thanks Wael, two minor comments and you are good to complete it.

File: docs/aca/07-aca-cron-bindings/index.md (hunk 2 above)
Comment (waelkdouh) [reply docs/aca/07-aca-cron-bindings/index.md:128]: @tjoudeh can you please take care of this one as I don't have a working version of the Newtonsoft implementation. I will go ahead as this is not a blocking change. Thanks

**Pull request merged**